design-system/src/design-tokens.ts

```diff
@@ -1,38 +1,295 @@
+    constraintScale: '100%',
+    constraint1: '42px',
+    constraint2: '84px',
+    constraint3: '142px',
+    constraint4: '184px',
+    constraint5: '242px',
+    constraint6: '284px',
+    constraint7: '342px',
+    constraint8: '384px',
+    constraint9: '442px',
+    constraint10: '484px',
+    constraint11: '542px',
+    constraint12: '584px',
+    constraint13: '642px',
+    constraint14: '684px',
+    constraint15: '742px',
+    constraint16: '784px',
+    spacingXs: '4px',
+    spacingS: '8px',
+    spacingM: '16px',
+    spacingL: '24px',
+    spacingXl: '32px',
+    transitionLinear80Ms: '80ms linear',
+    transitionEaseinout150Ms: '150ms ease-in-out',
+    transitionStandard: '200ms ease',
+    breakPointMobile: '768px',
+    breakPointDesktop: '1024px',
+    breakPointBiggerdesktop: '1280px',
+    breakPointGiantdesktop: '1680px',
+    breakPointJumbodesktop: '1920px',
+    backgroundColorForInput: '#fff',
+    backgroundColorForInputWhenSelected: 'hsl(195, 35.2941176471%, 95%)',
+    backgroundColorForInputWhenDisabled: 'hsl(195, 35.2941176471%, 98%)',
+    backgroundColorForInputWhenHovered: 'hsl(0, 0%, 90%)',
+    backgroundColorForTag: 'hsl(0, 0%, 95%)',
+    backgroundColorForTagWarning:
+      'hsl(25.110132158590307, 89.0196078431%, 95%)',
+    borderColorForInput: 'hsl(0, 0%, 60%)',
+    borderColorForInputWhenFocused: '#00b39e',
+    borderColorForInputWhenDisabled: '#ccc',
+    borderColorForInputWhenReadonly: '#ccc',
+    borderColorForInputWhenError: '#e60050',
+    borderColorForInputWhenWarning: '#f16d0e',
+    borderColorForTag: 'hsl(0, 0%, 60%)',
+    borderColorForTagWarning: '#f16d0e',
+    borderColorForTagWhenFocused: '#00b39e',
+    borderRadiusForInput: '6px',
+    borderRadiusForTag: '2px',
+    fontColorForInput: '#1a1a1a',
+    fontColorForInputWhenDisabled: 'hsl(0, 0%, 60%)',
+    fontColorForInputWhenError: '#e60050',
+    fontColorForInputWhenReadonly: 'hsl(0, 0%, 60%)',
+    fontColorForInputWhenWarning: '#f16d0e',
+    fontColorForTag: '#1a1a1a',
+    fontColorForTagWhenDisabled: 'hsl(0, 0%, 60%)',
+    fontColorForTextWhenInverted: '#fff',
+    fontFamily: "'Open Sans', sans-serif",
+    placeholderFontColorForInput: 'hsl(0, 0%, 60%)',
+    fontSizeForInput: '1rem',
+    shadowBoxTagWhenHovered:
+      '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
+    bigButtonHeight: '32px',
+    smallButtonHeight: '24px',
+    sizeHeightInput: '32px',
+    fontFamilyDefault: "'Open Sans', sans-serif",
+    fontSizeDefault: '1rem',
+    fontSizeSmall: '0.9231rem',
+    shadowBoxTagHover:
+      '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
+    sizeHeightTag: '26px',
+    standardInputHeight: '32px',
+  },
+  dark: {
+    colorPrimary: '#f78d26',
+    colorSolid: '#fff',
+    colorSurface: '#1a1a1a',
+    backgroundColorForTag: 'hsl(25.110132158590307, 89.0196078431%, 95%)',
+    backgroundColorForTagWarning: 'hsl(0, 0%, 95%)',
+  },
```

Comment:
What do we plan to do with this? Not sure we should merge this...or at least have a different name?

Reply:
I'm not sure either. As this is only used [here](https://github.com/commercetools/ui-kit/blob/292cb29e0a2bc98570ba4f8c7eac137017362bad/design-system/src/theme-provider.spec.js#L108) and [here](https://github.com/commercetools/ui-kit/blob/292cb29e0a2bc98570ba4f8c7eac137017362bad/design-system/src/theme-provider.spec.js#L143), I renamed it to `test`. I think @CarlosCortizasCT [is in favour of leaving it be](https://github.com/commercetools/ui-kit/pull/2227/commits/0dca49ff90dfcfbb4d4c40903ac92e412a444e17).